app/services/registration_service.rb

```diff
@@ -1,7 +1,10 @@
 class RegistrationService
   def from_invitation(email, invitation_token, user_data = {})
-    Invitation.find_by(email: email, token: invitation_token)
-    User.create({ email: email }.merge(user_data))
+    invitation = Invitation.find_by(email: email, token: invitation_token)
+    user = User.where(email: email).first
```

Comment:
I know it, but I like them to be raised in almost all the cases. It's not like AR where find_by are not raising exceptions, but find. So you can choose whether you want 'em or not.
If you want we can avoid model find exceptions and change where it was being used.
cc\ @gponsu